cdp_scrapers/instances/portland.py

```diff
@@ -239,13 +319,35 @@ def get_votes(self, event_page: BeautifulSoup) -> Optional[List[Vote]]:
         # Clearly “Yea” -> VoteDecision.APPROVE, but we don’t have enough information
         # for what words to map to other constants.
 
-        return reduced_list(
-            [
-                self.get_none_if_empty(
-                    Vote(decision=None, person=self.get_person(None))
-                ),
-            ],
+        vote_element_uncle = event_page.find(
+            "div", text=re.compile("Votes"), attrs={"class": "field__label"}
         )
+        if vote_element_uncle is None:
+            return None
+        vote_element_parent = vote_element_uncle.next_sibling
+        vote_elements = vote_element_parent.findAll("div", class_="relation--type-")
```

Comment:
I think we should use `find_all()` for consistency sake. Or change the other calls to `findAll()`. Either way.

Reply:
Done.